### PR TITLE
lynis: init at 2.5.7

### PR DESCRIPTION
**Pull request by ryneeverett**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/tools/security/lynis/default.nix [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  src = fetchFromGitHub {
+    owner = "CISOfy";
+    repo = "${pkgName}";
+    rev = "${version}";
+    sha256 = "19rfkiri73bi43i4yxpqrxjzpqn5rfrkq2picja5filjv14hbyly";
+  };
+
+  buildInputs = [ makeWrapper perl ];
+
+  patchPhase = ''
```
Comment (FRidh) [inline pkgs/tools/security/lynis/default.nix]: `postPatch` so as not to break the possibility of passing in patches

**Comment by ryneeverett** [comment]
Amended to satisfy review.

File: pkgs/tools/security/lynis/default.nix [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{ stdenv, makeWrapper, fetchFromGitHub, gawk, perl }:
+
+stdenv.mkDerivation rec {
+  pkgName = "lynis";
```
Comment (FRidh) [inline pkgs/tools/security/lynis/default.nix]: Note that `pname` is commonly used.

**Comment by ryneeverett** [comment]
Amended to satisfy review.

**Pull request merged**